submissions/rubyAdeniregun/php/orderpage.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+require_once("db_connection.php");
+
+echo "<h5> Order Page </h5>";
+
+setcookie("name", "json_encode($result)", time() + (86400 * 30), "/");
+
+try{
+    $sql= "SELECT * FROM order_details";
```

Comment:
The query you are to write for this task affects about 3 to 4 tables. That's why you were thought things like relationships on tables using foreign keys and the use of `join` statement